checkov/cloudformation/runner.py

```diff
@@ -63,5 +70,42 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                                                 evaluations=variable_evaluations,check_class=check.__class__.__module__,
                                                 file_abs_path=file_abs_path, entity_tags=tags)
                                 report.add_record(record=record)
+
+        if self.should_create_graph():
+            graph_report = self.get_graph_checks_report(root_folder, runner_filter)
+            merge_reports(report, graph_report)
+
         return report
 
+    def get_graph_checks_report(self, root_folder, runner_filter: RunnerFilter):
+        local_graph = self.graph_manager.build_graph_from_definitions(self.definitions)
```

Comment:
_local_graph_ is needed in the _run_ method in order to create the definitions from vertices and use them in the python checks, so I suggest to put the _build_graph_ and _save_graph_ in the run method

Reply:
let's create a `self.cfn_definitions` - same as the tf runner, and use it in both places

Reply:
@nimrodkor I created `self.definitions` which are the same as `cfn_definitions`, but I agree with saar, the build and save graph should be in `run`, the same as in terraform